src/components/Login.tsx

```diff
@@ -71,7 +71,7 @@ const Login = ({ loginWalletOptions }: LoginProps): JSX.Element => {
 
   useEffect(() => {
     ethereum?.on("accountsChanged", () => logout());
-  }, []);
+  }, [logout]);
```

Comment:
Blocking: I'm not sure what prompted this, but it isn't right. Logout gets defined every render, so this method will get called constantly, and we'll keep redefining the subscription. That's a guaranteed memory leak. I'm not sure when the ethereum subscription got added, but this isn't the right place for that either.